src/DropDown/DropDown.component.tsx

```diff
@@ -60,48 +84,54 @@ const StyledDropDownContainer = styled(DropDownContainer)<
 >`
     min-width: 10rem;
     position: absolute;
-    padding: 0.25rem 0;
+    padding: 0.25rem auto;
     z-index: 1;
     background-color: ${th.color('neutrals.white.base')};
```

Comment:
Would be nice to get props for some of these.

Reply:
It would, but it present a ton of challenges making it that composable and I don't think we'll need that yet. 80/20 argument, which is super weak...